client/app/views/recipient/tips.html

```diff
@@ -126,6 +127,7 @@
                 <span data-ng-show="sortReverse" class="fa-solid fa-caret-up"></span>
               </span>
             </span>
+            <span translation-texts="dropdownDefaultText"  ng-dropdown-multiselect="" options="dropdownStatusData" selected-model="dropdownStatusModel" events="on_changed"  extra-settings="dropdownSettings"> </span>
```

Comment:
ng-dropdown-multiselect need to be rewritten data-ng-dropdown-multiselect in relation to wai-aria.

probably also al the other directives used miss the "data-" prefix